draft-ietf-quic-recovery.md

```diff
@@ -216,23 +215,23 @@ implemented in QUIC.
 
 An unacknowledged packet is marked as lost when an acknowledgment is received
 for a packet that was sent a threshold number of packets (kReorderingThreshold)
-after the unacknowledged packet. Receipt of the ack indicates that a later
-packet was received, while kReorderingThreshold provides some tolerance for
-reordering of packets in the network.
-
-The RECOMMENDED initial value for kReorderingThreshold is 3.
-
-We derive this recommendation from TCP loss recovery {{?RFC5681}}
-{{?RFC6675}}. It is possible for networks to exhibit higher degrees of
-reordering, causing a sender to detect spurious losses. Detecting spurious
-losses leads to unnecessary retransmissions and may result in degraded
-performance due to the actions of the congestion controller upon detecting
-loss. Implementers MAY use algorithms developed for TCP, such as TCP-NCR
-{{?RFC4653}}, to improve QUIC's reordering resilience, though care should be
-taken to map TCP specifics to QUIC correctly. Similarly, using time-based loss
-detection to deal with reordering, such as in PR-TCP, should be more readily
-usable in QUIC. Making QUIC deal with such networks is important open research,
-and implementers are encouraged to explore this space.
+and/or a threshold amount of time after the unacknowledged packet. Receipt of
+the acknowledgement indicates that a later packet was received, while the
+reordering threshold provides some tolerance for reordering of packets in the
+network.
+
+The RECOMMENDED initial value for kReorderingThreshold is 3, based on
+TCP loss recovery {{?RFC5681}} {{?RFC6675}}. Some networks may exhibit higher
+degrees of reordering, causing a sender to detect spurious losses. Spuriously
+declaring packets lost leads to unnecessary retransmissions and may result in
+degraded performance due to the actions of the congestion controller upon
+detecting loss. Implementers MAY use algorithms developed for TCP, such as
+TCP-NCR {{?RFC4653}}, to improve QUIC's reordering resilience.
+
+QUIC implementations may use time-based loss detection to deal with reordering.
+time-based loss detection allows a certain amount of reordering above
+max(SRTT, latest_rtt).  The RECOMMENDED reordering tolerance is
+1/8 max(SRTT, latest_rtt).
```

Comment:
This should be reworded. How about: "QUIC implementations can use time-based loss detection to handle reordering based on time elapsed since the packet was sent, instead of a packet reordering threshold. The RECOMMENDED time threshold, expressed as a fraction of the round-trip time (kTimeReorderingFraction), is 1/8." (The details of SRTT, etc are in the algo below, so I think being generic is ok here.)